src/Microsoft.Diagnostics.Monitoring.RestServer/Models/EventPipeProvider.cs

```diff
@@ -9,8 +9,9 @@
 
 namespace Microsoft.Diagnostics.Monitoring.RestServer.Models
 {
-    [DataContract(Name = "EventPipeProvider")]
-    public class EventPipeProviderModel
+    [DataContract]
+    public class EventPipeProvider
```

Comment:
Any chance we're going to have an api naming clash with other types here?

Reply:
Yes, they do clash with the ones under Microsoft.Diagnostics.NETCore.Client only if the Microsoft.Diagnostics.Monitoring.RestServer.Models is used. So I've just used these model classes with the Model. namespace qualifier since these are largely just used in the controllers.